.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,49 @@
+name: CI Workflow
+
+on:
+  [push, workflow_dispatch]
+
+concurrency: ci-${{ github.ref }}
+
+jobs:
+  buildAndTest:
+    name: Build, Test, and Package
+    runs-on: ubuntu-latest
+    env:
+      RESOURCE_GROUP: hri-dev1-wdc-kube
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install IBM Dev Tools & DevOps Insights CLI plug-in
+        run: |
+          curl -sL https://ibm.biz/idt-installer | bash
+          ibmcloud plugin install doi
+
+      - name: Log into IBM Cloud
+        shell: bash
+        env:
+          CLOUD_API_KEY: ${{ secrets.CLOUD_API_KEY }}
+        run: |
+          IBMCLOUD_API_KEY="$CLOUD_API_KEY" ibmcloud login -r us-south || { echo 'IBM Cloud CLI login failed!'; exit 1; }
+          ibmcloud target -g ${RESOURCE_GROUP}
```

Comment:
I don't think we actually have to do the two above steps (Install IBM Dev Tools & DevOps Insights CLI plug-in and Log into IBM Cloud) anymore. I'm guessing that was only needed either for the previous "run-insights-publish" script or for publishing to artifactory -- either way I think they can go.

Reply:
I believe @dan-baxter is correct. Any objections to removing this?

Reply:
I removed the section and the build still passes.